Directory.Build.targets

```diff
@@ -24,12 +24,13 @@
     <Compile Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.cs" Link="Common\System\PlatformDetection.cs" />
     <Compile Condition="'$(OS)' != 'Windows_NT'" Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.Unix.cs" Link="Common\System\PlatformDetection.Unix.cs" />
     <Compile Condition="'$(OS)' == 'Windows_NT'" Include="$(CommonPath)tests\TestUtilities\System\PlatformDetection.Windows.cs" Link="Common\System\PlatformDetection.Windows.cs" />
```

Comment:
As mentioned in the other comment, these three files should be merged into a single one and always compiled, irrelevant of which host OS we are on. Using platform runtime detection makes more sense here.